js/views/pageNavVw.js

```diff
@@ -501,11 +501,13 @@ module.exports = baseVw.extend({
   },
 
   navBackClick: function(){
-    window.history.back();
+    history.action = 'back';
```

Comment:
This approach assumes that the only way that back / forward navigation will occur is via these buttons. That is not always the case. It may be necessary to programmatically navigate (e.g. History.back()), which I believe we are doing in at least one place, and after doing so your 'position' calculation will be off.